### DISCUSSION TITLE
webadv Refresh_Token 变动导致无法访问网盘

### DISCUSSION THREAD

**Discussion by woaimi520** [messense/aliyundrive-webdav#369] [Q&A]
问题描述
       apple-tv 通过安装梅林固件的路由器 使用webAdv方式访问如阿里的云盘，z按配置配置好refresh_token，且能读取到
网盘内容，但是一旦我在手机端登录网盘，refresh token就会过期，导致apple tv 无法访问网盘。 

期望
     能有机制定时更新token

**Comment by messense** [comment]
没遇到过，请提供日志。

**Reply by qwer123wd** [reply]
我也碰到了，我是安装了 小白羊第三方阿里云APP，登录后，nas重启 nas上的阿里云webdev token就过期必须重新抓取在填新的token
